### ISSUE TITLE
results.plot_corner() does not correctly interpret the param_list variable

### ISSUE THREAD

**Issue by adj-smith** [sblunt/orbitize#234]
When using results.plot_corner() and defining a param_list variable to restrict the posteriors displayed, the resulting corner plot is not produced / labelled correctly. It appears to only take note of the first term (? further testing requried) when producing the corner plot. However, it correctly pulls the symbols for each parameter and orders them by the order given. This results in mislabeled plots.

Example:

```
corner_figure_median_95 = m.results.plot_corner(
	param_list = ['sma1', 'ecc1', 'inc1', 'aop1', 'pan1', 'tau1'],
	show_titles = True
)
```

produces
![orbits47Uma060421](https://user-images.githubusercontent.com/22928268/120864478-82a3f600-c549-11eb-92af-d30f7aeb2b84.png)

while

```
corner_figure_median_95 = m.results.plot_corner(
	truths=median_values,
	param_list = ['tau1','sma1','pan1','aop1','ecc1','inc1'],
	show_titles = True
)
```

instead produces
![orbits47Uma060321](https://user-images.githubusercontent.com/22928268/120864587-ab2bf000-c549-11eb-9bff-bad380e43e5e.png)

Note in particular the third variable: in the first image, this is labelled as the position angle of nodes, while in the second it is labelled as the inclination, yet the posterior is the same in both images.

**Comment by sblunt** [comment]
Hi Adam! Thanks so much for the bug report. I actually can't reproduce this bug with the latest version of orbitize. Are you using an older version? If so, let me know if you still see this behavior after updating.

**Comment by adj-smith** [comment]
Hi Sarah,

I was in fact using an older version (thanks for reminding me to upgrade!) - however, even after upgrading to 1.15.3 the issue still persists. This **might** be a local issue, possibly related to my OS (Windows 10). I've had a number of other problems with orbitize! which mysteriously vanish when I upload my code to a remote, Unix-operating computing system.

**Comment by semaphoreP** [comment]
Just to check, @adj-smith , are you recalculating the posterior after you updated to the latest version of orbitize! or are you reading in a old posterior from disk? 

It looks like just inc and pan are being confused? The rest seem ok.

**Comment by adj-smith** [comment]
@semaphoreP I did not recalculate the posterior after upgrading today. The posterior was calculated on a version from 1-2 months ago. I'm planning to do an overnight retrieval tonight.

Since I can't run ptemcee on my Win10 Python3 setup without a bunch of fiddling, I have two setups - the remote Unix server that I run my retrievals on and my local Windows box that I do my analysis on. At some point the two got out of sync with each other (until today, when I updated both of them).

**Comment by semaphoreP** [comment]
Ok, please check whether the problem persists if you recalculate the posterior with the latest orbitize code. The bug you encountered likely arose when the posteriors were created, and we may have fixed that bug in the meantime.

**Comment by semaphoreP** [comment]
@adj-smith I think I fixed it in PR #236. Take a look at the example there and see if that seems satisfactory!

**Comment by sblunt** [comment]
Addressed in PR #236
